public/js/apps/management/app.jsx

```diff
@@ -1,20 +1,33 @@
 import 'shims';
 
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
 import { Provider, Connector } from 'redux/react';
 import { bindActionCreators } from 'redux';
 
 import reduxConfig from 'redux-config';
 import * as managementActions from 'actions/management';
 import * as userActions from 'actions/user';
+import { parseQuery } from 'utils';
 
 import ModalError from 'views/modal-error';
-import Management from 'views/management';
-import ManageCards from 'views/manage-cards';
+import { default as DefaultManagement } from 'views/management';
```

Comment:
I think you could just do `DefaultManagement from blah` but this is probably nicer as it's more explicit.